tests/foreman/ui/test_errata.py

```diff
@@ -396,86 +332,129 @@ def test_end_to_end(
         ],
         'module_stream_packages': [],
     }
-    _UTC_format = '%Y-%m-%d %H:%M:%S UTC'
-    # Capture newest product and repository with the desired content
-    product_list = module_target_sat.api.Product(organization=module_org).search()
-    assert len(product_list) > 0
-    product_list.sort(key=lambda product: product.id)
-    _product = product_list[-1].read()
-    assert len(_product.repository) == 1
-    _repository = _product.repository[0].read()
-    # Remove custom package if present, install outdated version
-    registered_contenthost.execute(f'yum remove -y {FAKE_1_CUSTOM_PACKAGE_NAME}')
-    result = registered_contenthost.execute(f'yum install -y {FAKE_1_CUSTOM_PACKAGE}')
-    assert result.status == 0, f'Failed to install package {FAKE_1_CUSTOM_PACKAGE}.'
-    # recalculate and assert applicable errata after installing outdated pkg
-    assert registered_contenthost.execute('subscription-manager repos').status == 0
-    applicable_errata = registered_contenthost.applicable_errata_count
+    # client was registered with single custom repo
+    client = registered_contenthost
+    hostname = client.hostname
+    assert client.subscribed
+    custom_repo = module_cv.read().repository[0].read()
+    # nothing applicable to start
+    result = client.execute('subscription-manager repos')
+    assert (
+        result.status == 0
+    ), f'Failure invoking subscription-manager content, for client: {hostname}.\n{result.stdout}'
+    assert (
+        0 == client.applicable_errata_count == client.applicable_package_count
+    ), f'Expected no applicable erratum or packages to start, on host: {hostname}'
+    # install outdated package version, making an errata applicable
+    result = client.execute(r'subscription-manager repos --enable \*')
+    assert (
+        result.status == 0
+    ), f'Failed to enable available repositories with subscription-manager.\n{result.stdout}'
```

Comment:
Instead of `--enable \*`, could we use `enable_repos=True` from the needed_setup used in the `registered_contenthost` [here](https://github.com/SatelliteQE/robottelo/pull/15085/files#diff-bd6863972772cca482646a04c83fba2589bdb45b1fcfdb0c4d544cd9fd17c920R213)?

Reply:
Yes, I wanted to test both methods in Errata UI, but do you think we should just stick to enabling through AK in the api_factory method?

Reply:
I would think so. Looks more like the customer approach to me - define the AK properly first and then just use it.

Reply:
Yes since we already parameterize `registered_contenthost` fixture with the repo URL, 
it is already enabled in AK and `api_factory`, we don't need to enable in this test.